include/exadg/fluid_structure_interaction/precice/precice_adapter.h

```diff
@@ -115,9 +117,13 @@ class Adapter
                    std::vector<std::string> const &                               read_data_name);
 
   /**
-   * @brief      Advances preCICE after every timestep
+   * @brief Advances preCICE after every timestep
+   *
+   * @param computed_timestep_length the time-step size computed by the solver
+   *
+   * @return double the allowed time-step size until the next coupling time-window is reached
```

Comment:
related to another comment above: What is the difference between "the allowed time-step size until ..." or simply "the time until ..."?

Reply:
They are the same, I changed the terminology.